### DISCUSSION TITLE
Support for C bitfields in structs

### DISCUSSION THREAD

**Discussion by H-A-M-G-E-R** [NationalSecurityAgency/ghidra#7233] [Ideas]
example:
```
struct A {
    byte a:1;
    byte b:7;
};
```

**Comment by H-A-M-G-E-R** [comment]
Many GBA games use these, example: https://github.com/H-A-M-G-E-R/mf/blob/main/include/oam.h#L70

**Comment by gemesa** [comment]
https://github.com/NationalSecurityAgency/ghidra/issues/647

**Comment by H-A-M-G-E-R** [comment]
i didn't know that existed
